src/autowiring/marshaller.h

```diff
@@ -171,104 +153,66 @@ namespace autowiring {
   };
 
   template<typename T>
+  struct float_converter;
+
+#ifdef MSVC
```

Comment:
In that case you may want to make the condition to be `defined(_MSC_VER) && _MSC_VER <= 1900`